### ISSUE TITLE
difference result with different version mmdetection3d

### ISSUE THREAD

**Issue by AndyYuan96** [open-mmlab/mmdetection3d#1752]
Hi, I want to try transfusion with mmdet3d 1.0, and transfusion's repo use mmdet3d 0.11, and I copy the transfusion head and boxencoder to mmdet3d 1.0. But I can't reproduce the result with 1/5 data, when use mmdet3d 1.0, the performance drop about 2 point in map and nds. after some experiment, I find when I use init weight from mmdet3d 0.11 to train transfusion with mmdet3d 1.0, the performance don't drop. So I just don't why init function will have so much difference on detector, so I just wonder whether your guys have any advise?
the init function change comes from here: https://github.com/open-mmlab/mmdetection3d/pull/378

**Comment by ZCMax** [comment]
To unify the parameter initialization in OpenMMLab projects, MMCV supports `BaseModule` that accepts `init_cfg` to allow the modules' parameters initialized in a flexible and unified manner. Now the users need to explicitly call `model.init_weights()` in the training script to initialize the model (as in [here](https://github.com/open-mmlab/mmdetection3d/blob/master/tools/train.py#L183), previously this was handled by the detector. Please refer to PR [#622](https://github.com/open-mmlab/mmdetection3d/pull/622) for details. Did you use `BaseModule` instead of `nn.Module` while migrating the transfusion head and boxencoder to mmdet3d 1.0?

**Comment by AndyYuan96** [comment]
> To unify the parameter initialization in OpenMMLab projects, MMCV supports `BaseModule` that accepts `init_cfg` to allow the modules' parameters initialized in a flexible and unified manner. Now the users need to explicitly call `model.init_weights()` in the training script to initialize the model (as in [here](https://github.com/open-mmlab/mmdetection3d/blob/master/tools/train.py#L183), previously this was handled by the detector. Please refer to PR [#622](https://github.com/open-mmlab/mmdetection3d/pull/622) for details. Did you use `BaseModule` instead of `nn.Module` while migrating the transfusion head and boxencoder to mmdet3d 1.0?

no, I don't use base module instead of nn.module. and I think it's not related to that, as after I change the default init cfg=None of second and secondfpn, and use mmdet3d 0.11's init_weight function, the loss curve looks right, it's almost the same with mmdet3d 0.11's loss curve. but with mmdet3d 1.0's init_cfg of second and second fpn, the loss curve is always higher than mmdet3d0.11.

**Comment by BoomSky0416** [comment]
@AndyYuan96 Hello, I encountered the same performance drop during reproducing transfusion in the latest mmdet3d. Could you  show me how to modify the initialization of second and secondfpn? Thanks!

**Comment by xtk8532704** [comment]
I also encountered the same problem when reproducing transfusion.
